music21/musicxml/m21ToXml.py

```diff
@@ -3892,15 +3891,27 @@ def noteToNotations(self, n, notFirstNoteOfChord=False, chordParent=None):
                 else:
                     notations.append(mxExpression)
 
-            for artObj in chordOrNote.articulations:
-                if 'TechnicalIndication' in artObj.classes:
-                    if mxTechnicalMark is None:
-                        mxTechnicalMark = Element('technical')
-                    mxTechnicalMark.append(self.articulationToXmlTechnical(artObj))
-                else:
-                    if mxArticulations is None:
-                        mxArticulations = Element('articulations')
-                    mxArticulations.append(self.articulationToXmlArticulation(artObj))
+        # apply all articulations apart from fingerings only to first note of chord
+        from music21.articulations import Fingering
+        applicableArticulations = []
+        fingeringNumber = 0
+        for a in chordOrNote.articulations:
+            if type(a) is Fingering:
+                if fingeringNumber == pitchNumber:
+                    applicableArticulations.append(a)
+                fingeringNumber += 1
+            elif pitchNumber == 0:
+                applicableArticulations.append(a)
```

Comment:
what if there are more Fingering articulations than pitches? will they be lost?

Reply:
this is otherwise really good!

Reply:
yes they will be intentionally lost. I added that behavior to doc and test